rules/eslint/suggestions.js

```diff
@@ -0,0 +1,213 @@
+module.exports = {
+  rules: {
+    'accessor-pairs': 'error',
+    'arrow-body-style': ['error', 'as-needed'],
+    // handled by "no-var" rule
+    'block-scoped-var': 'off',
+    camelcase: ['error', { properties: 'always' }],
+    'capitalized-comments': ['error', 'always', { ignoreConsecutiveComments: true }],
```

Comment:
Is it possible to configure this rule so that it won't replace `/* prettier-ignore */` with `/* Prettier-ignore */` and so on?

Reply:
That should be feasible by providing a custom `ignorePattern` property:

```
ignorePattern: ["jscs", "jshint", "eslint", "istanbul", "global", "globals", "exported", "prettier"]
```

Reply:
Another downside is that the rule capitalizes the commented out code which doesn't look right:

```diff
- // await oneEvent(chart, 'chart-redraw');
+ // Await oneEvent(chart, 'chart-redraw');
```

Reply:
Let's disable this rule. It's not always required to have the comment capitalized.